src/ui/view/dataverse/component/filters/filter.scss

```diff
@@ -0,0 +1,48 @@
+@import '@/ui/style/fonts.scss';
+@import '@/ui/style/theme.scss';
+@import '@/ui/style/mixins.scss';
+
+.okp4-dataverse-portal-dataverse-filter-main {
+  @include with-theme {
+    position: relative;
+    padding: 8px 0 16px;
+
+    &:not(:last-child)::after {
+      content: '';
+      opacity: 0.8;
+      position: absolute;
+      bottom: 0;
+      left: 0;
+      right: 0;
+      width: 100%;
+      height: 1px;
+      background-image: radial-gradient(
+        rgba(themed('text'), 0.2) 0%,
+        rgba(themed('text'), 0) 80%
+      );          
+    }
+
+    .okp4-dataverse-portal-dataverse-filter-trigger {
+      background-color: themed('background');
+    }
+  
+    .okp4-dataverse-portal-dataverse-filter {
+      @include columns-with-gap(16px);
+      padding-top: 8px;
+    }
+  }
+}
+
+.okp4-dataverse-portal-dataverse-filter-label {
```

Comment:
this one shouldn't be inside `okp4-dataverse-portal-dataverse-filter-trigger` ?

Reply:
Nope because it is also used in the context of [FiltersChips](https://github.com/okp4/dataverse-portal/blob/d42fa0044558467acc5de2c6923597ca614d043d/src/ui/page/dataverse/dataverse.tsx#L691).